DotnetFoundation/DotnetFoundation.Application/Models/Common/PagingRequest.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace DotnetFoundation.Application.Models.Common;
+public class PagingRequest
+{
+    const int MaxPageSize = 50;
+    public int PageNumber { get; set; }
+    private int pageSize;
```

Comment:
Why does it have to be in camel casing? Is there a standard that we are following?

Reply:
This is being used to get the PageSize below

Reply:
It is a private variable. As per standards it should be cameCase preceeded with `_`.
Fixed in latest PR